index.html

```diff
@@ -466,347 +466,365 @@ <h3>Ethereum EIP712 Signature 2021</h3>
-                }
-              ],
-              "CredentialSubject": [
-                {
-                  "name": "type",
-                  "type": "string"
-                },
+        <p>
+          With the following <code>inputOptions</code> provided to the signature suite along with the <code>testBasicDocument</code> input document:
+        </p>
+
+        <pre class="example">
+          {
+            "date": "2021-08-30T13:28:02Z"
+          }
+        </pre>
+
+        <p>The following is the resulting <code>proof</code> object:</p>
+
+        <pre class="example">
+          {
+            "created": "2021-08-30T13:28:02Z",
+            "proofPurpose": "assertionMethod",
+            "type": "EthereumEip712Signature2021",
+            "verificationMethod": "did:pkh:eth:0xAED7EA8035eEc47E657B34eF5D020c7005487443",
+            "proofValue": "0xd8ced27b921866a9cb6fb859503714ad4be03ae70706237d05c9f113da7f4a1d7f74f9f9df4301571f5c4f253c09e1b5119a85f00760f33924d72a07d31881ec1b",
+            "eip712Domain": {
+              "domain": {},
+              "messageSchema": {
+                "Document": [
+                  {
+                    "name": "@context",
+                    "type": "string[]"
+                  },
+                  {
+                    "name": "@type",
+                    "type": "string"
+                  },
+                  {
+                    "name": "email",
+                    "type": "string"
+                  },
+                  {
+                    "name": "firstName",
+                    "type": "string"
+                  },
+                  {
+                    "name": "jobTitle",
+                    "type": "string"
+                  },
+                  {
+                    "name": "lastName",
+                    "type": "string"
+                  },
+                  {
+                    "name": "telephone",
+                    "type": "string"
+                  }
+                ]
+              },
+              "primaryType": "Document"
+            }
+          }
+        </pre>
+      </section>
+      <section>
+        <h3>Nested Document - TypedData Provided - Embedded Types</h3>
+
+        <p>
+          With the following <code>inputOptions</code> provided to the signature suite along with the <code>testNestedDocument</code> input document:
+        </p>
+
+        <pre class="example">
+          {
+            "types": {
+              "Data": [
                 {
-                  "name": "id",
-                  "type": "string"
+                  "name": "job",
+                  "type": "Job"
                 },
                 {
                   "name": "name",
-                  "type": "string"
-                },
-                {
-                  "name": "child",
-                  "type": "Person"
+                  "type": "Name"
                 }
               ],
-              "EIP712Domain": [
```

Comment:
Isn't the `EIP712Domain` property required by EIP-712?

Reply:
Hmm, @oed what do you think?

Reply:
I don't see why we wouldn't comply with the spec.

Reply:
We can default to something like
```
domain: {
    name: "EthereumEip712Signature2021"
}
```